src/lib/cfa/sudoers.rb

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast2/target_file"
+require "cfa/base_model"
+require "cfa/augeas_parser"
+require "cfa/matcher"
+require "cfa/placer"
+
+module CFA
+  # Model to handle users specifications in the /etc/sudoers file
+  #
+  # @example Loading specifications
+  #   file = Sudoers.new
+  #   file.load
+  #   file.current_user_specs
+  #
+  # @example Shortcut loading specificitions
+  #   Sudoers.load.users_specs
```

Comment:
this does not work, as result of load is `true` and not model itself. See https://github.com/config-files-api/config_files_api/blob/master/lib/cfa/base_model.rb#L58
and it is also documented, that result is void, so you should not depends on it https://github.com/config-files-api/config_files_api/blob/master/lib/cfa/base_model.rb#L49

Reason is quite simple. This shortcut is very ineffecient if you want to use it multiple times.

Reply:
ah, now I see that you have your own load, so let me comment it close to that method.

Reply:
> ah, now I see that you have your own load, so let me comment it close to that method.

:innocent: :innocent: 

To be honest, I took this idea borrowed from the recent @imobachgs's work [adding a CFA model to handle shadow configuration](https://github.com/yast/yast-yast2/pull/988). I liked it and I think is useful for the "Read" operation.